src/main/kotlin/no/nav/tilleggsstonader/sak/vedlegg/VedleggController.kt

```diff
@@ -0,0 +1,20 @@
+package no.nav.tilleggsstonader.sak.vedlegg
+
+import no.nav.security.token.support.core.api.ProtectedWithClaims
+import no.nav.tilleggsstonader.sak.tilgang.AuditLoggerEvent
+import no.nav.tilleggsstonader.sak.tilgang.TilgangService
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/api/vedlegg")
+@ProtectedWithClaims(issuer = "azuread")
+class VedleggController(private val tilgangService: TilgangService, private val vedleggService: VedleggService) {
+    @PostMapping("/fagsak-person")
```

Comment:
Burde vi ha med `fagsakPersonId` i path? då får vi vi med fagsak-personen i access-logging for den requesten :)
```suggestion
    @PostMapping("/fagsak-person/{fagsakPersonId}")
```

Reply:
Vi kan også vurdere om vi på et aller annet sted skal legge inn slik informasjon i MDC, sånn at det blir logger i alle logger i stedet for kun i access-loggen.
Eks at man i controllers legger inn (litt forenklet)
`MDC.put("fagsakPersonId", fagsakPersonId)`, då blir fagsakPersonId logget i alle logger for den requesten som et felt, likt `callId`